nebula/ui/components.qrc

```diff
@@ -111,5 +111,7 @@
         <file>components/inAppAuth/VPNInAppAuthenticationLegalDisclaimer.qml</file>
         <file>components/inAppAuth/VPNInAppAuthenticationCancel.qml</file>
         <file>components/forms/VPNCursorDelegate.qml</file>
+        <file>components/VPNSegmentedToggle.qml</file>
```

Comment:
✨ **Segmented Toggle** ✨ ... I was wondering what this little piece of navigation should be called and segmented toggle seems to be exactly correct.